ros_buildfarm/common.py

```diff
@@ -42,6 +42,30 @@ def __init__(self, message):  # noqa: D107
         super(JobValidationError, self).__init__(message)
 
 
+class RepositoryPackageDescriptor(str):
```

Comment:
Not sure I like the name - but also short of good alternative proposals since it don't know what else is planned to added... Maybe `BinaryPackageDescriptor`?

Reply:
5c01b1b